src/main/scala/com/microsoft/hyperspace/index/IndexSignatureProvider.scala

```diff
@@ -30,9 +30,10 @@ import com.microsoft.hyperspace.util.HashingUtils
  * If the plan does not comply with [[FileBasedSignatureProvider]] or [[PlanSignatureProvider]]
  * requirements for signature computation, then no signature will be provided for the plan.
  */
-class IndexSignatureProvider extends LogicalPlanSignatureProvider {
-  private val fileBasedSignatureProvider = new FileBasedSignatureProvider
-  private val planSignatureProvider = new PlanSignatureProvider
+class IndexSignatureProvider(fbf: FingerprintBuilderFactory)
```

Comment:
Please update the scaladoc. It should contain the `@param fbf` explaining its role.

Reply:
Done